app/controllers/uploads_controller.rb

```diff
@@ -23,10 +23,10 @@ def destroy
     @upload = Upload.find(params[:id])
     @upload.destroy
     if current_user.admin?
-      flash[:danger] = "File deleted."      
+      flash[:danger] = "File deleted."
       redirect_to admin_dashboard_path
     else  
-      flash[:danger] = "File deleted."            
+      flash[:danger] = "File deleted."
```

Comment:
@cmacaulay I think this is just showing us that he removed the trailing whitespace after the string.